swift/src/Ice/LocalException.swift

```diff
@@ -1,1030 +1,47 @@
-
-    /// Returns a string representation of this exception
-    ///
-    /// - returns: `String` - The string representaton of this exception.
-    override open func ice_print() -> String {
-        return _MarshalExceptionDescription
-    }
-}
-
-/// This exception is raised if an unsupported feature is used. The unsupported feature string contains the name of the
-/// unsupported feature.
-open class FeatureNotSupportedException: LocalException {
-    /// The name of the unsupported feature.
-    public var unsupportedFeature: String = ""
-
-    public required init() {
-        super.init()
-    }
-
-    public init(unsupportedFeature: String, file: String = #file, line: Int32 = #line) {
-        self.unsupportedFeature = unsupportedFeature
-        super.init(file: file, line: line)
-    }
-
-    /// Returns the Slice type ID of this exception.
-    ///
-    /// - returns: `String` - the Slice type ID of this exception.
-    override open class func ice_staticId() -> String {
-        return "::Ice::FeatureNotSupportedException"
-    }
-
-    /// Returns a string representation of this exception
-    ///
-    /// - returns: `String` - The string representaton of this exception.
-    override open func ice_print() -> String {
-        return _FeatureNotSupportedExceptionDescription
-    }
-}
-
-/// This exception indicates a failure in a security subsystem, such as the IceSSL plug-in.
-open class SecurityException: LocalException {
-    /// The reason for the failure.
-    public var reason: String = ""
-
-    public required init() {
-        super.init()
-    }
-
-    public init(reason: String, file: String = #file, line: Int32 = #line) {
-        self.reason = reason
-        super.init(file: file, line: line)
-    }
-
-    /// Returns the Slice type ID of this exception.
-    ///
-    /// - returns: `String` - the Slice type ID of this exception.
-    override open class func ice_staticId() -> String {
-        return "::Ice::SecurityException"
-    }
-
-    /// Returns a string representation of this exception
-    ///
-    /// - returns: `String` - The string representaton of this exception.
-    override open func ice_print() -> String {
-        return _SecurityExceptionDescription
-    }
-}
-
-/// This exception indicates that an attempt has been made to change the connection properties of a fixed proxy.
-open class FixedProxyException: LocalException {
-    /// Returns the Slice type ID of this exception.
-    ///
-    /// - returns: `String` - the Slice type ID of this exception.
-    override open class func ice_staticId() -> String {
-        return "::Ice::FixedProxyException"
-    }
-
-    /// Returns a string representation of this exception
-    ///
-    /// - returns: `String` - The string representaton of this exception.
-    override open func ice_print() -> String {
-        return _FixedProxyExceptionDescription
+/// Base class for Ice local exceptions.
+open class LocalException: Exception, CustomStringConvertible {
```

Comment:
Should it be `public class` instead? 

I kept `open class` since that what we have in 3.7.

Reply:
`public class` seems more correct now. I suspect we needed `open` in 3.7 for local slice.